ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -44,16 +42,6 @@ sub prepare_action
         }
     }
 
-    if ($srv->{daemon}) {
```

Comment:
Fair enough, but how would this code get exposed to old templates? If the component is update the profile must be rebuilt and the schema validation would throw an error.